modules/compliance-profilebundle.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-manage.adoc
+
+[id="compliance-profilebundle_{context}"]
+= Creating a ProfileBundle
```

Comment:
I would expect a procedure module here, given the heading. I think you can make a procedure and then follow it with the example. Alternatively, you can rename the module "ProfileBundle Example" or something like that if you think this would be better as conceptual material.